addons/mail/static/src/discuss/call/common/call_participant_card.scss

```diff
@@ -71,6 +88,18 @@
 
 .o-discuss-CallParticipantCard-overlay {
     margin: Min(5%, map-get($spacers, 2));
+
+    &.o-proportional-container {
+        container-type: inline-size;
+        .o-proportional-text {
+            &.o-video-text {
+                font-size: 5cqw;
+            }
+            &.o-live-text {
+                font-size: 8cqw;
+            }
```

Comment:
This reduces the font-size too much, to the point the "live" text is not visible.
<img width="228" alt="Screenshot 2025-02-19 at 13 23 06" src="https://github.com/user-attachments/assets/adbb87c5-3c4c-425e-8c17-21df49dee6cc" />

The label should increase its width and keep text readable instead, something like this:
<img width="232" alt="Screenshot 2025-02-19 at 13 23 31" src="https://github.com/user-attachments/assets/31f21122-3f8d-46be-8ff1-2277cf5930d8" />